metomi/rose/etc/tutorial/cylc-forecasting-workflow/.validate

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
```

Comment:
We don't really want this, but oh well

Reply:
Probably easy enough to sed out?

On the other hand, why not be honest about the origin of this stuff - @oliver-sanders  seems to have done some reading baout copyright, so I'd be interested in hearing his view.

Reply:
Will need to check this for legal reasons, please raise at team meeting.